File: Keyboard Input/ADumbCalculator.java
import java.util.Scanner;

public class ADumbCalculator
{
	public static void main( String[] args )
	{
		Scanner keyboard = new Scanner(System.in);

		System.out.print( "What is your first number? " );
		int number1 = keyboard.nextInt();

		System.out.print( "What is your second number?" );
		int number2 = keyboard.nextInt();

        System.out.print( "What is your third number?" );
        int number3 = keyboard.nextInt();

        double total = number1+number2+number3;
	    System.out.print( "( "+number1+" + "+number2+" + "+number3+" )/2 is..."+total/2 );
		

       
	}
}

File: Keyboard Input C++/ADumbCalculator.cpp
#include<iostream>

using namespace std;

int main()
{
    cout<<"What is your first number? \n" ;
    int a;
    cin >>a;
    cout<<"What is your second number? \n" ;
    int b;
    cin >> b;
    cout<<"What is your third number? \n" ;
    int c;
    cin >>c;

    int d=a+b+c;

    cout<<"( 1.1 + 2.2 + 5.5 ) / 2 is... "<< d*0.5 <<"\n" ;
}